app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -62,6 +62,11 @@ angular.module('Bastion.components').factory('Nutupane',
             params.page = $location.search().page || 1;
             params['per_page'] = $location.search()['per_page'] || entriesPerPage;
 
+            //Additional param to override initial load
+            self.overrideAutoLoad = params.overrideAutoLoad || false;
```

Comment:
Hey @ehelms , @waldenraines : 
In an earlier discussions with Eric, we discussed an alternate approach of using the action parameter to go about this. Would that be the preferred approach? Overriding the self.load() through the params is probably the approach with least impact elsewhere, hence I proceeded this way.

( I am not familiar with the masterOnly property although I did see it in a few controllers)